omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/ConditionController2_2Test.java

```diff
@@ -325,4 +327,30 @@ public void shouldReturnAllConditions() throws Exception {
 		List<Condition> conditions = result.get("results");
 		Assert.assertEquals(3, conditions.size());
 	}
+
+	@Test
+	public void shouldGetACustomRepresentation() throws Exception {
+		MockHttpServletRequest request = request(RequestMethod.GET, getURI());
+		request.addParameter("patientUuid", "da7f524f-27ce-4bb2-86d6-6d1d05312bd5");
+		request.addParameter("includeInactive", "true");
+		request.addParameter("v", "custom:(uuid,display,clinicalStatus,onsetDate,endDate,additionalDetail,condition:(coded:(id,uuid)))");
+		SimpleObject result = deserialize(handle(request));
```

Comment:
Without the fix, this test fails here.  It throws an error trying to process this request.